renderdoc/driver/gl/gl_driver.cpp

```diff
@@ -777,10 +787,25 @@ WrappedOpenGL::~WrappedOpenGL()
 
 ContextPair &WrappedOpenGL::GetCtx()
 {
-  ContextPair *ret = (ContextPair *)Threading::GetTLSValue(m_CurCtxPairTLS);
+  GLContextTLSData *ret = (GLContextTLSData *)Threading::GetTLSValue(m_CurCtxDataTLS);
   if(ret)
-    return *ret;
-  return m_EmptyPair;
+    return ret->ctxPair;
+  return m_EmptyTLSData.ctxPair;
+}
+
+GLResourceRecord *WrappedOpenGL::GetContextRecord()
+{
+  GLContextTLSData *ret = (GLContextTLSData *)Threading::GetTLSValue(m_CurCtxDataTLS);
+  if(ret && ret->ctxRecord)
+  {
+    return ret->ctxRecord;
+  }
+  else
+  {
+    ContextData dat = GetCtxData();
```

Comment:
It's minor because this is an emergency fallback path, but this is copying ContextData and creating the resource record there instead of creating it in the original - this should be `ContextData &dat`.